### DISCUSSION TITLE
Get request for `email_auth_request`?

### DISCUSSION THREAD

**Discussion by benkoshy** [jeremyevans/rodauth#248] [General]
* is it a ridiculous idea for there to be a get route for `email_auth_request`? for someone new to rod_auth, I was expecting this route to be a get request where the auth_email form would be displayed.
* to promote internal consistency with naming. e.g. `password_reset_request` (is the latter a get / post request?) as suggested by @janko 
* Genesis:  from the [rodauth-rails library discussion](https://github.com/janko/rodauth-demo-rails/discussions/10#discussioncomment-3016889)

**Comment by jeremyevans** [comment]
It's not a ridiculous idea, but I'm not sure what the need is.  When using the email_auth feature, you login using the normal login form.  If you only have email auth configured, it sends an email.  If you have other methods of authentication, it shows you your possible options.  

I'm not sure what value a separate form for only email auth could add.  The argument for it seems to be consistency with other Rodauth features (e.g. reset_password), but it misses the point that the other forms exist for a reason (e.g. the "Forgot Password?" link in the reset_password case), which would not be true for this form.

You mentioned in the rodauth-rails discussion: `I think the fundamental cause is that the rake routes task doesn't show the HTTP verb, with any parameters, as rails does`.  That seems to be a rodauth-rails issue, not a Rodauth issue.
